staging/operator-registry/go.sum

```diff
@@ -1066,6 +1066,7 @@ honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc/go.mod h1:rf3lG4BRIbNafJWh
 honnef.co/go/tools v0.0.1-2019.2.3/go.mod h1:a3bituU0lyd329TUQxRnasdCoJDkEUEAqEt0JzvZhAg=
 honnef.co/go/tools v0.0.1-2020.1.3/go.mod h1:X/FiERA/W4tHapMX5mGpAtMSVEeEUOyHaw9vFzvIQ3k=
 honnef.co/go/tools v0.0.1-2020.1.4/go.mod h1:X/FiERA/W4tHapMX5mGpAtMSVEeEUOyHaw9vFzvIQ3k=
+<<<<<<< HEAD
```

Comment:
Need to fix this in this commit

Reply:
Or ... not because it's a straight upstream pick?

Reply:
Yikes, thanks!

Reply:
I did need to fix this commit. You have to run `go mod tidy` in the downstream project as the dependencies may differ as:
- Some dependencies differ, since not all features are backported.
- Some of the downstream projects, particularly OLM, have downstream specific dependencies.